features/inventory.feature

```diff
@@ -27,6 +27,22 @@ Scenario: Retrieve an Inventory
     And I should see "100" in the "Restock Level" field
     And I should see "True" in the "Available" dropdown
 
+Scenario: Add amount to an Inventory
+    When I visit the "Home Page"
+    And I set the "Product ID" to "1"
+    And I select "NEW" in the "Condition" dropdown
+    And I set the "Add_amount" to "200"
```

Comment:
We can use do

```
And I set the "Add amount" to "200"
```

since we have

```
def __get_element_id(element_name: str) -> str:
    return 'inventory_' + "_".join(element_name.lower().split())
```

in `features/steps/web_steps.py`

as long as the `id` of each element follow the same pattern.